docs/virtual_machines/disks_and_volumes.md

```diff
@@ -412,6 +412,56 @@ spec:
         claimName: mypvc
 ```
 
+
+#### block device from host
+
```

Comment:
Here, are missing a short introduction why we need statically PVC. You could mentioned, your use-case to share a disk from the host to the VM